app/controllers/questionnaire_controller.rb

```diff
@@ -3,6 +3,8 @@
 class QuestionnaireController < ApplicationController
   MAX_ANSWER_LENGTH = 2048
   include Concerns::IsLoggedIn
+  protect_from_forgery prepend: true, with: :exception, except: :create
```

Comment:
wordt dit niet ergens in de `ApplicationController` ook aangeroepen? en zoja, moet hij daar dan ook niet uit / toegevoegd worden aan de andere subclasses?

Reply:
Ja dit wordt ook in de application controller aangeroepen. Hier overschrijf ik die aanroep voor deze klasse (en volgens mij ook voor klasses die hiervan ervan maar die zijn er niet).